src/writeConfluence.js

```diff
@@ -0,0 +1,58 @@
+const { markdown } = require('markdown')
+const Confluence = require('confluence-api')
+
+const config = {
+    username: process.env.CONFLUENCE_USER,
+    password: process.env.CONFLUENCE_API_KEY,
+    baseUrl:  "https://ecobee.atlassian.net/wiki"
```

Comment:
Please remove all ecobee specific URLs. It's a public repo

Reply:
Will do. Forgot this was public.